sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -775,4 +775,16 @@ class UDFSuite extends QueryTest with SharedSparkSession {
     }
     assert(e2.getMessage.contains("UDFSuite$MalformedClassObject$MalformedPrimitiveFunction"))
   }
+
+  test("SPARK-32307: Aggression that use map type input UDF as group expression") {
```

Comment:
Aggression ? Maybe Aggregation?

Reply:
yea should be Aggregation

Reply:
I am writing tests for intervals in UDF now. I will fix this, if you don't mind.

Reply:
thanks!

Reply:
Here is the PR with the fix: https://github.com/apache/spark/pull/31779